apps/foundation-app/src/app/cart/components/cart/cart.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+
+import { CartFactory } from '@daffodil/core';
+
+import { CartComponent } from './cart.component';
+
+describe('CartComponent', () => {
+  let component: CartComponent;
+  let fixture: ComponentFixture<CartComponent>;
+  let cartFactory = new CartFactory();
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ CartComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CartComponent);
+    component = fixture.componentInstance;
+
+    component.carts = new Array(cartFactory.create(), cartFactory.create());
```

Comment:
same thing here, a cart is a single object, not an array.

Reply:
changed